string.md

```diff
@@ -0,0 +1,54 @@
+---
+title: STRING
+toc: true
+---
+
+## Description
+
+The `STRING` [data type](data-types.html) stores strings of variable, unlimited length. 
+
+## Synonyms 
+
+In CockroachDB, the following are synonyms of `STRING` and are implemented identically: 
+
+- `CHAR` 
+- `CHAR(n)` 
+- `VARCHAR`
+- `VARCHAR(n)` 
+- `TEXT`
+
+## Length
+
+Length is always variable and unlimited for a `STRING` value. If you specify a fixed length by declaring `CHAR(n)` or `VARCHAR(n)` on a column, it will not be enforced.
```

Comment:
The length is supposed to be supported. That we're not handling this correctly for columns is a bug. Consider:

```
root@:26257> select 'hello world'::char(5);
+--------------------------------+
| CAST('hello world' AS CHAR(5)) |
+--------------------------------+
| hello                          |
+--------------------------------+
```